packages/jest-haste-map/src/ModuleMap.ts

```diff
@@ -66,13 +62,13 @@ export default class ModuleMap {
 
   getPackage(
     name: string,
-    platform: ?string,
-    supportsNativePlatform: ?boolean,
```

Comment:
not sure why we pass `null` to `getModule` instead of using this argument

Reply:
yea, just wanted to comment about it. maybe we should start passing it?

Reply:
IDK. @rubennorte?